dev/AppNotifications/AppNotificationUtility.cpp

```diff
@@ -215,6 +215,20 @@ HRESULT Microsoft::Windows::AppNotifications::Helpers::GetActivatorGuid(std::wst
 }
 CATCH_RETURN()
 
+std::wstring Microsoft::Windows::AppNotifications::Helpers::SetDisplayNameBasedOnProcessName()
+{
+    std::wstring displayName{};
+    THROW_IF_FAILED(wil::GetModuleFileNameExW(GetCurrentProcess(), nullptr, displayName));
+
+    size_t lastBackslashPosition{ displayName.rfind(L"\\") };
+    THROW_HR_IF(E_UNEXPECTED, lastBackslashPosition == std::wstring::npos);
+    displayName = displayName.substr(lastBackslashPosition + 1); // One after the delimiter
+
+    displayName.erase(displayName.find_first_of(L".")); // Remove file extension
+
```

Comment:
Make sure that the display name starts with caps: For example Tiktok and not tiktok.